index.js

```diff
@@ -99,16 +143,59 @@ function createRenderer() {
 }
 
 /**
+ * Get service iframe video by passed name
+ * Required href:
+ * - youtube: https://youtu.be/1GWoMnYldYc or https://www.youtube.com/watch?v=1GWoMnYldYc
+ * - yandex: https://video.yandex.ru/iframe/ya-events/clg24o8tu3.7046/
+ * - vimeo: https://vimeo.com/30838628 or https://player.vimeo.com/video/30838628
+ * @param {String} href - full video href, examples see above
+ * @param {String} serviceName - name of the service [youtube, yandex, vimeo]
+ * @returns {String} - iframe video block
+ */
+function getIframe(href, serviceName) {
+    var serviceToUrl = {
+            youtube: 'www.youtube.com/embed/',
+            yandex: 'video.yandex.ru/iframe/ya-events/',
+            vimeo: 'player.vimeo.com/video/'
+        },
+        arr = href.split('/'),
+        key = arr.pop(),
+        src;
+
+    /**
+     * Get the second last href arr item for key,
+     * if href contains slash at the end and our key empty
+     * For example: https://vimeo.com/53219242/
+     */
+    if (!key.length) {
```

Comment:
I am tried to use this module https://www.npmjs.com/package/oembed-node and then created an issue https://github.com/hgarcia/oEmbed-node/issues/7 and I thing this module has many unnecessary code.

I researched other modules and in all cases, I must register on some web service, get the API key, in some cases I cant take only iframe and others. So, I think, for out solution we shouldn't want third-party modules